Meta/Lagom/ReadMe.md

```diff
@@ -22,7 +22,8 @@ the ``BuildFuzzers.sh`` script with no arguments. The script does the equivalent
     # Stage 1: Build and install code generators and other tools
     cmake -GNinja -B Build/tools \
       -DBUILD_LAGOM=OFF \
-      -DCMAKE_INSTALL_PREFIX=Build/tool-install
+      -DCMAKE_INSTALL_PREFIX=Build/tool-install \
+      -Dpackage=LagomTools
```

Comment:
:thinking: IMO we just shouldn't list the exact `cmake` commands these fuzzers run. The script itself is the source of truth,

Reply:
The script doesn't work on macOS and this worked though. (Ish -- it doesn't link, but it works well enough to check if a fuzzer compiles. Which was enough for me to add a fuzzer yesterday.)

Reply:
Right, my point is that this doc should just say something like "if you need to build manually, check out *this part* of the script for the cmake incantations". It doesn't make sense to have to copy-paste commands from the script to this readme.

(Of course the script itself should be fixed to work on macOS, but that's a separate issue)

Reply:
My point is that that's also something different from what this PR is doing. I saw that
1. The doc is wrong
2. The build script isn't working
3. The whole thing seems overly complicated (when I added the doc in the first place long ago, iirc it was much simpler)

I figured I can spend the minute to improve the docs. I tried looking into 2 for another minute but that took longer than a minute.

If you think it's better to have the steps in there and have them incorrect rather than having in there and have them at least work until someone rewrites the doc (…or breaks the steps again), feel free to close ofc. imho this is a (tiny) net positive though, so I made a PR for it. (No hard feelings if you think it's not worth it.)